wiki/Tournaments/OFFC/en.md

```diff
@@ -0,0 +1,5 @@
+# osu! FumoFumo Cup
+
+Index page for all of the osu! FumoFumo Cup series.
+
+- [osu! FumoFumo Cup cirno](cirno)
```

Comment:
Should have the colon as that is shown as the heading of the main tournament article:
```suggestion
- [osu! FumoFumo Cup: cirno](cirno)
```